### DISCUSSION TITLE
Cant override Content-Type headers in vercel.json

### DISCUSSION THREAD

**Discussion by screendriver** [vercel/community#2266] [Help]
I'm trying to override `Content-Type` headers in the [headers section](https://vercel.com/docs/concepts/projects/project-configuration#headers) of my `vercel.json`. But it looks like that `Vercel` is ignoring it or at least overriding it with its own values.

Here is an example of my [vercel.json](https://vercel.com/docs/concepts/projects/project-configuration):

```json
{
  "headers": [
    {
      "source": "/audio/(.+).(webm|aac)",
      "headers": [
        {
          "key": "Cache-Control",
          "value": "public, max-age=604800, immutable"
        }
      ]
    },
    {
      "source": "/audio/(.+).webm",
      "headers": [
        {
          "key": "Content-Type",
          "value": "audio/webm"
        }
      ]
    },
    {
      "source": "/audio/(.+).aac",
      "headers": [
        {
          "key": "Content-Type",
          "value": "audio/aac"
        }
      ]
    }
  ]
}
```

What I try to express is that all audio files in my `/audio/` directory has the correct `Content-Type` header set. But instead of `audio/webm` Vercel is serving `video/webm`. Interestingly the `Cache-Control` header is working as expected but not `Content-Type`.

I also tried different source expressions like `/audio\/(.+).webm` or `/audio/(.+)\\.webm` and so on but none of them worked so I guess `Vercel` is overriding these values with its own values.

**Comment by amyegan** [comment]
Hi @screendriver! I wasn't able to reproduce this issue in my test app. My configured content-type headers came through as planned, even on files for which the headers make no sense. 
<details>
<summary>My vercel.json config</summary>

```json
{
  "headers": [
    {
      "source": "/(.+).(json)",
      "headers": [
        {
          "key": "Content-Type",
          "value": "audio/webm"
        }
      ]
    },
    {
      "source": "/(.+).txt",
      "headers": [
        {
          "key": "Content-Type",
          "value": "audio/aac"
        }
      ]
    }
  ]
}
```
</details>

Could there be something changing the header elsewhere in the code? Maybe an imported package is setting `video/webm`

**Reply by screendriver** [reply]
Hi @amyegan,

thanks for your help. You are right. I deployed my `vercel.json` to `Vercel` and it's indeed working as expected when it's running in your infrastructure. But not when running locally via `vercel dev`. Usually I try things out locally first so I didn't knew that it will work when I deploy it to `Vercel`.

So I think this is still a bug in the `vercel` CLI that it is not working locally.
